app/src/lib/oauth.ts

```diff
@@ -67,9 +67,10 @@ export async function requestAuthenticatedUser(
  */
 export function resolveOAuthRequest(account: Account) {
   if (!oauthState) {
-    return fatalError(
+    fatalError(
       '`askUserToOAuth` must be called before resolving an auth request.'
     )
+    return
```

Comment:
If `fatalError` throws an `Error`, what's the need to return here? Is it just a TS/JS thing?

Reply:
Yeah, I think it's because `fatalError` technically returns `never` where the rest of the function returns `void`.